charts/rancher-backup/files/default-resourceset-contents/fleet.yaml

```diff
@@ -19,6 +19,9 @@
 - apiVersion: "v1"
   kindsRegexp: "^configmaps$"
   namespaceRegexp: "^cattle-fleet-|^fleet-|^cluster-fleet-"
+- apiVersion: "scheduling.k8s.io/v1"
```

Comment:
Right now this will get all priorityClasses - we only want to backup the one that the rancher chart specifies. You can do this with a `resourceNameRegexp` field, like [here](https://github.com/rancher/backup-restore-operator/blob/7e8dc2f00dad7879160be4f0bff2ec55b27fb215/charts/rancher-backup/files/default-resourceset-contents/fleet.yaml#L3). 

I would recommend anchoring the start and the end so that we exactly match what [rancher's chart](https://github.com/rancher/rancher/blob/a841a9e101ae601a033fa8b39342ad4800c91d85/chart/templates/priorityClass.yaml#L4) defines  - so something like `resourceNameRegexp: "^rancher-critical$"`